src/pages/workspace/receiptPartners/EditInviteReceiptPartnerPolicyPage.tsx

```diff
@@ -0,0 +1,334 @@
+        (email: string): UberEmployeeStatus => {
+            const status = uberEmployeesByEmail[email]?.status?.toLowerCase();
+            switch (status) {
+                case 'accepted':
+                    return CONST.POLICY.RECEIPT_PARTNERS.UBER_EMPLOYEE_STATUS.LINKED;
+                case 'invited':
+                    return CONST.POLICY.RECEIPT_PARTNERS.UBER_EMPLOYEE_STATUS.INVITED;
+                case 'pending_approval':
+                case 'accepted_pending_approval':
+                    return CONST.POLICY.RECEIPT_PARTNERS.UBER_EMPLOYEE_STATUS.LINKED_PENDING_APPROVAL;
+                case 'suspended':
+                    return CONST.POLICY.RECEIPT_PARTNERS.UBER_EMPLOYEE_STATUS.SUSPENDED;
+                case 'deleted':
+                    return CONST.POLICY.RECEIPT_PARTNERS.UBER_EMPLOYEE_STATUS.DELETED;
+                default:
+                    return CONST.POLICY.RECEIPT_PARTNERS.UBER_EMPLOYEE_STATUS.NONE;
+            }
+        },
+        [uberEmployeesByEmail],
+    );
+
+    const members = useMemo<Array<MemberForList & ListItem>>(() => {
+        const list: Array<MemberForList & ListItem> = [];
+        const employees = policy?.employeeList ?? {};
+        Object.entries(employees).forEach(([email, policyEmployee]) => {
+            // Skip deleted policy employees
+            if (isDeletedPolicyEmployee(policyEmployee, isOffline)) {
+                return;
+            }
+            const personalDetail = getPersonalDetailByEmail(email);
+            const option = formatMemberForList({
+                text: personalDetail?.displayName ?? email,
+                alternateText: email,
+                login: email,
+                accountID: personalDetail?.accountID,
+                icons: [
+                    {
+                        source: personalDetail?.avatar ?? Expensicons.FallbackAvatar,
+                        name: formatPhoneNumber(email),
+                        type: CONST.ICON_TYPE_AVATAR,
+                        id: personalDetail?.accountID,
+                    },
+                ],
+                keyForList: email,
+                reportID: '',
+            });
+            const status = deriveStatus(email);
+            let badgeText: string | undefined;
+            switch (status) {
+                case CONST.POLICY.RECEIPT_PARTNERS.UBER_EMPLOYEE_STATUS.LINKED:
+                    badgeText = translate('workspace.receiptPartners.uber.linked');
+                    break;
+                case CONST.POLICY.RECEIPT_PARTNERS.UBER_EMPLOYEE_STATUS.LINKED_PENDING_APPROVAL:
+                    badgeText = translate('workspace.receiptPartners.uber.pending');
+                    break;
+                case CONST.POLICY.RECEIPT_PARTNERS.UBER_EMPLOYEE_STATUS.SUSPENDED:
+                    badgeText = translate('workspace.receiptPartners.uber.suspended');
+                    break;
+                default:
+                    break;
```

Comment:
Isn't this missing more statuses?
Also, instead of doing a case, why not do something like:
```
badgeText = translate(`workspace.receiptPartners.uber.${status}`);
```
?